lib/precompiled-abi-forks-util.js

```diff
@@ -13,6 +13,7 @@ const IRIS = 'iris';
 const HOP = 'hop';
 const HOP401 = 'HOP401';
 const FINGERROOT = 'fingerroot';
+const ARROWHEAD = 'tbd600';
```

Comment:
Not possible yet. This value is sent as part of the configuration for the nodes. This value is not yet replaced in the rskj project, if I replace it here, it will not match and from the rskj perspective, it will not get activated or will take a default value set in rskj if present.